palace/models/curlcurloperator.hpp

```diff
@@ -69,6 +69,9 @@ class CurlCurlOperator
   auto &GetRTSpace() { return rt_fespace; }
   const auto &GetRTSpace() const { return rt_fespace; }
 
+  // Return the number of true (conforming) dofs on the finest ND space.
+  auto GetNDof() { return GetNDSpace().GlobalTrueVSize(); }
```

Comment:
Can we just call these `GlobalTrueVSize` to be consistent with the MFEM `FiniteElementSpace` naming?